types/serverless/classes/Service.d.ts

```diff
@@ -15,6 +15,11 @@ declare class Service {
       };
 
       name: string;
+      stage: string;
+      region: string;
+      runtime?: string;
+      timeout?: number;
+      versionFunctions: boolean;
```

Comment:
Some of these new properties are non-optional, are they really required?

Reply:
Hey gabritto! My methodology was to print the service object on the command line when those options were not supplied. For some options, when not provided, there are no members in the Service object; for others, there were.

For instance, I made `region` non-optional, because, even when a region is not specified in `serverless.yml`, there is still a `region` property in the Service element, which is filled in with the appropriate value.

Now that you mention it, though, I do wonder if this is different with different providers. I don't know if Azure has a concept of regions...